### PR TITLE
Implement express-heroku

### PR DESCRIPTION
**Pull request by preveenraj**
Implemented express with heroku.
@vishnuprasad-95 @ajayesivan

### CODE REVIEW

**Review by vishnuprasad-95 — APPROVED** [review]

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: node server/bin/www
```
Comment (ajayesivan) [inline Procfile:1]: What is this file for? What does this command do?

File: assets.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"main":{"js":"bundle.js"}}
```
Comment (ajayesivan) [inline assets.json:1]: What is this file for?

File: package.json [hunk 3]
```diff
@@ -1,49 +1,63 @@
 {
-  "name": "bookshelf",
+  "name": "wheel.js",
```
Comment (ajayesivan) [inline package.json:2]: The project name should be bookshelf

File: package.json [hunk 4]
```diff
@@ -1,49 +1,63 @@
...
   "version": "1.0.0",
-  "description": "",
   "main": "index.js",
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
+  "repository": "https://github.com/bigbinary/wheel.js.git",
```
Comment (ajayesivan) [inline package.json:5]: Change repo & author

File: package.json [hunk 5]
```diff
@@ -1,49 +1,63 @@
...
+    "@babel/preset-env": "^7.10.2",
+    "@babel/preset-react": "^7.10.1",
+    "assets-webpack-plugin": "^5.0.0",
+    "babel-loader": "^8.1.0",
+    "case-sensitive-paths-webpack-plugin": "^2.3.0",
+    "webpack": "^4.43.0",
+    "webpack-cli": "^3.3.11",
+    "webpack-dev-server": "^3.11.0"
   },
-  "husky": {
```
Comment (ajayesivan) [inline package.json:39]: Why removing husky?

**Review by ajayesivan — CHANGES_REQUESTED** [review]

File: assets.json (hunk 2 above)
Comment (preveenraj) [reply assets.json:1]: it is an autogenerated file with the help of [AssetsPlugin](https://www.npmjs.com/package/assets-webpack-plugin)
Why is it used for? To dynamically generate paths of assets in our app in a json file.
and we can use that json file anywhere like our views embed the bundle into it.
@ajayesivan

File: package.json (hunk 5 above)
Comment (preveenraj) [reply package.json:39]: removed by mistake, brought back. thanks for the care observation.

File: assets.json (hunk 2 above)
Comment (ajayesivan) [reply assets.json:1]: Why we need an autogenerated file in git?

Comment (preveenraj) [reply assets.json:1]: yes, but without that file, for the first time a developer runs the app, his app will crash. He should stop the app and run it again to make it work. Thats a cumbersome step. Do we need to ignore it still? @ajayesivan

Comment (ajayesivan) [reply assets.json:1]: Git's not a place for generated files. So I feel you should remove it from git & do something to generate it when building the project and if that's not possible we can keep it.

Comment (ajayesivan) [reply assets.json:1]: @preveenraj I will merge now. Any way you check how to properly handle it.

**Review by ajayesivan — APPROVED** [review]

File: assets.json (hunk 2 above)
Comment (preveenraj) [reply assets.json:1]: odikko kandam vazhi. sheri nokaam @ajayesivan

**Pull request merged**